osu.Android.props

```diff
@@ -10,11 +10,7 @@
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="ppy.osu.Framework.Android" Version="2022.1216.1" />
-  </ItemGroup>
-  <ItemGroup Label="Transitive Dependencies">
-    <!-- Realm needs to be directly referenced in all Xamarin projects, as it will not pull in its transitive dependencies otherwise. -->
-    <PackageReference Include="Realm" Version="10.18.0" />
+    <PackageReference Include="ppy.osu.Framework.Android" Version="2022.1219.0" />
   </ItemGroup>
```

Comment:
Has Android been checked with this removed? Slightly hesitant about it.

Reply:
yep i tested on a device